.github/workflows/pr-hub.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       CORDE_BOT_TOKEN: ${{ secrets.CORDE_BOT_TOKEN }}
     services:
       postgres:
-        image: postgres:13
+        image: postgres:14
```

Comment:
Is this part of the fix?

Reply:
Yes, I've tried using version 13 and the CLI failed.

Reply:
I can't see how increasing pg version would solve this problem